docs/sources/set-up/install/windows.md

```diff
@@ -52,44 +53,66 @@ To do a silent install of {{< param "PRODUCT_NAME" >}} on Windows, perform the f
+- `/ENVIRONMENT="KEY=VALUE\0KEY2=VALUE2"` Define environment variables for Windows Service. Default: ``
+- `/RUNTIMEPRIORITY="normal|below_normal|above_normal|high|idle|realtime"` Set the runtime priority of the {{< param "PRODUCT_NAME" >}} process. Default: `normal`
+- `/STABILITY="generally-available|public-preview|experimental"` Set the stability level of {{< param "PRODUCT_NAME" >}}. Default: `generally-available`
+- `/USERNAME="<username>"` Set the fully qualified user that Windows uses to run the service. Default: `NT AUTHORITY\LocalSystem`
+- `/PASSWORD="<password>"` Set the password of the user that Windows uses to run the service. This isn't required for standard Windows Service Accounts like LocalSystem. Default: ``
 
 {{< admonition type="note" >}}
-The `--windows.priority` flag is in [Public preview][stability] and is not covered by {{< param "FULL_PRODUCT_NAME" >}} [backward compatibility][] guarantees.
-The `/RUNTIMEPRIORITY` installation option sets this flag, and if Alloy is not running with an appropriate stability level it will fail to start.
+The `--windows.priority` flag is in [public preview][stability] and isn't covered by {{< param "FULL_PRODUCT_NAME" >}} [backward compatibility][] guarantees.
+The `/RUNTIMEPRIORITY` installation option sets this flag, and if {{< param "PRODUCT_NAME" >}} isn't running with an appropriate stability level, it fails to start.
 
 [stability]: https://grafana.com/docs/release-life-cycle/
 [backward compatibility]: ../../../introduction/backward-compatibility/
+
 {{< /admonition >}}
 
-## Service Configuration
+## Service configuration
 
-{{< param "PRODUCT_NAME" >}} uses the Windows Registry `HKLM\Software\GrafanaLabs\Alloy` for service configuration.
+{{< param "PRODUCT_NAME" >}} uses the Windows registry key `HKLM\Software\GrafanaLabs\Alloy` for service configuration.
 
-* `Arguments` (Type `REG_MULTI_SZ`) Each value represents a binary argument for alloy binary.
-* `Environment` (Type `REG_MULTI_SZ`) Each value represents a environment value `KEY=VALUE` for alloy binary.
+- `Arguments`: Type `REG_MULTI_SZ`. Each value represents a binary argument for the {{< param "PRODUCT_NAME" >}} binary.
+- `Environment`: Type `REG_MULTI_SZ`. Each value represents an environment value `KEY=VALUE` for the {{< param "PRODUCT_NAME" >}} binary.
 
 ## Uninstall
 
-You can uninstall {{< param "PRODUCT_NAME" >}} with Windows Add or Remove Programs or `%PROGRAMFILES%\GrafanaLabs\Alloy\uninstall.exe`.
 Uninstalling {{< param "PRODUCT_NAME" >}} stops the service and removes it from disk.
 This includes any configuration files in the installation directory.
 
-{{< param "PRODUCT_NAME" >}} can also be silently uninstalled by running `uninstall.exe /S` as Administrator.
+### Standard graphical uninstall
+
+To uninstall {{< param "PRODUCT_NAME" >}}, use Add or Remove Programs or run the following command as Administrator in PowerShell.
+
+```powershell
+%PROGRAMFILES%\GrafanaLabs\Alloy\uninstall.exe
+```
```

Comment:
This doesn't have to be done in PowerShell, and this isn't valid syntax for PowerShell.

For PowerShell the command would be:

```pwsh
& ${env:PROGRAMFILES}\GrafanaLabs\Alloy\uninstall.exe
```